src/applications/ezr/config/chapters/veteranInformation/emergencyContacts.js

```diff
@@ -0,0 +1,181 @@
+import merge from 'lodash/merge';
+import ezrSchema from 'vets-json-schema/dist/10-10EZR-schema.json';
+import { arrayBuilderPages } from '~/platform/forms-system/src/js/patterns/array-builder';
+import {
+  arrayBuilderYesNoSchema,
+  arrayBuilderYesNoUI,
+  addressUI,
+  addressSchema,
+  fullNameUI,
+  phoneUI,
+  selectUI,
+  titleUI,
+  yesNoUI,
+  yesNoSchema,
+} from '~/platform/forms-system/src/js/web-component-patterns';
+import content from '../../../locales/en/content.json';
+import { isEmergencyContactsEnabled } from '../../../utils/helpers/form-config';
+
+const {
+  veteranContacts: { items: contact },
+} = ezrSchema.properties;
+
+const { fullName, primaryPhone, relationship, address } = contact.properties;
+
+const arrayBuilderOptions = {
+  arrayPath: 'veteranContacts',
+  nounSingular: 'emergency contact',
+  nounPlural: 'emergency contacts',
+  required: false,
+  maxItems: 2,
+  isItemIncomplete: item =>
+    !item?.fullName?.first ||
+    !item?.fullName?.last ||
+    !item?.primaryPhone ||
+    !item?.relationship,
+  text: {
+    getItemName: item => {
+      if (item && item.fullName && item.fullName.first && item.fullName.last) {
+        return `${item.fullName.first} ${item.fullName.last}`;
+      }
+      return '';
+    },
+    cardDescription: item => `${item?.primaryPhone || ''}`,
+    deleteTitle: _ => 'Delete this emergency contact?',
+    deleteYes: _ => 'Yes, delete this emergency contact',
+    deleteDescription: item => {
+      if (item && item.fullName && item.fullName.first && item.fullName.last) {
+        return `This will delete ${item.fullName.first} ${
+          item.fullName.last
+        } and all the information from your list of emergency contacts.`;
+      }
+      return 'This will delete this contact and all the information from your list of emergency contacts.'; // Fallback if data is missing
+    },
+  },
+  depends: isEmergencyContactsEnabled,
+};
+
+const emergencyContactsPage = {
+  uiSchema: {
+    ...titleUI(
+      content['emergency-contact-title'],
+      content['emergency-contact-description'],
+    ),
+    fullName: {
+      ...fullNameUI,
```

Comment:
You don't have to redeclare the fields. You're essentially overwriting everything for this pattern. The fullNameUI gives you the opportunity to modify the labels to append a string--in this case "Emergency contact's". Review the `spouseFullName` or `dependentFullName`. Here is the pattern for spouse:

```
fullNameUI(title => `${content['household-spouse-name-prefix']} ${title}`),
```